src/char_mapping.rs

```diff
@@ -94,6 +164,50 @@ table! {
     }
 }
 
+table! {
+    #[doc = "A char-to-glyph encoding in format 12."]
+    pub Encoding12 {
+        format      (u16), // format
+        reserved    (u16) = { 0 }, // reserved
+        length      (u32), // length
+        language    (u32), // language
+        num_groups  (u32), // numGroups
+
+        groups (Vec<SequentialMapGroup>) |this, tape| { // groups
+            tape.take_given(this.num_groups as usize)
+        },
+    }
+}
+
+table! {
+    #[doc = "A char-to-glyph encoding in format 14."]
+    pub Encoding14 {
+        format               (u16), // format
+        length               (u32), // length
+        num_var_selectors    (u32), // numVarSelectorRecords
+
+        var_selector_records (Vec<VariationSelectorRecord>) |this, tape| { // varSelector
+            tape.take_given(this.num_var_selectors as usize)
+        },
+    }
+}
+
+table! {
+    #[doc = "A VariationSelectorRecord."]
+    pub VariationSelectorRecord {
+        var_selector (u32) |_, tape| { // varSelector
+            let buf: [u8; 3] = tape.take()?;
```

Comment:
Let’s call it `buffer`. As mentioned earlier, I try to avoid any kind of shortenings.